### PR TITLE
Added Javascript exercises from day 1 + day 2

### PR DESCRIPTION
**Pull request by sydneynguyen912**
Both javascript exercises finished in this PR.

### CODE REVIEW

**Comment by sydneynguyen912** [comment]
image
image

**Comment by sydneynguyen912** [comment]
image

**Comment by sydneynguyen912** [comment]
image



image

**Comment by sydneynguyen912** [comment]
Finished JS Udemy course: screenshot 
image

File: sydneynguyen/css/css/holygrail.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+.header, .footer {
+    background-color: rgb(185, 185, 185);
```
Comment (madhanramu) [inline sydneynguyen/css/css/holygrail.css:3]: use hex values for colors

File: sydneynguyen/css/css/holygrail.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    height: 100px;
+    width: 100%;
+    padding-bottom: 20px;
+  }
+  
+  .container {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
```
Comment (madhanramu) [inline sydneynguyen/css/css/holygrail.css:12]: flex is is the property of flex item. should not be using for container

File: sydneynguyen/css/css/holygrail.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    height: 300px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+  }
+  
+  .main {
+    background-color: rgb(171, 161, 181);
```
Comment (madhanramu) [inline sydneynguyen/css/css/holygrail.css:19]: use hex value instead of rgb.

File: sydneynguyen/js/js_findNearest.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/*Write a JavaScript program to find a value which is nearest to 
+100 from two different given integer values. Example Test cases:  
+findNearestValueTo100(80, 99) should result in 99 
+findNearestValueTo100(-10, -9) should result in -9 */
+
+function findNearestValueTo100() {
```
Comment (madhanramu) [inline sydneynguyen/js/js_findNearest.js:6]: good

**Review by madhanramu — CHANGES_REQUESTED** [review]

**Review by madhanramu — APPROVED** [review]

**Pull request merged**